keyboards/ymd75/keymaps/via/config.h

```diff
@@ -0,0 +1,3 @@
+#if DEVICE_VER == 0x0100
+	#define DYNAMIC_KEYMAP_LAYER_COUNT 3
```

Comment:
This goes back in the top-level config.h, without the `#if` block.

Reply:
You mean rev1 folder config.h is ignored that way?

Reply:
No, but this way all revisions have only 3 layers. This define must exactly match the number of layers in the VIA keymap.

However, if you want the other two revisions to still have 4 layers, then this define would go in the rev1 config.h, and each revision would have its own keymaps directory with its own `via` keymap with the appropriate number of layers. The top-level keymaps directory would still work, though, so you would not have to copy the existing keymaps into each revision. But as I said, it's not the cleanest solution.

Reply:
Ok, then 3 layers is enough.